blackbox/docs/installation.txt

```diff
@@ -1,105 +1,107 @@
 
-   sh$ tar zxf crate-*.tar.gz
+   sh$ tar -xzf crate-*.tar.gz
    sh$ cd crate-*
 
-Once unpacked CrateDB can be started in the foreground like this::
+Run
+===
+
+There is no need to configure or build anything.
+
+Once unpacked, CrateDB can be started in the foreground like this::
 
    sh$ ./bin/crate
 
-In order to start in the background add the ``-d`` option. If started in the
-foreground you can use ``Control-C`` to stop the process again.
+:kbd:`Control-C` will stop the process.
 
+Pass the ``--help`` option for command usage information::
 
-The CrateDB Shell (Crash) is shipped with the tarball distribution.
-You can start the shell using the ``crash`` command::
+   sh$ ./bin/crate --help
 
-  sh$ ./bin/crash
+.. SEEALSO::
 
-The CrateDB distribution also comes with a web based administration
-interface. It is served on port ``4200`` and you can access it with
-your browser once CrateDB is started::
+ :ref:`cli`
 
-  http://host1.example.com:4200/
+ :ref:`configuration`
 
-.. NOTE::
+Connect
+=======
 
-    The admin interface runs on each node where CrateDB is installed.
+The CrateDB Shell (aka `Crash`_) is shipped with the tarball distribution.
 
-.. NOTE::
+Start the shell like so::
 
-    If the previous URL is accessed via libraries (eg. cURL), or via offline
-    browsers (eg. wget) the request will be handled by the CrateDB Rest API,
-    and the response will be JSON.
+  sh$ ./bin/crash
 
-Install Using Docker
-====================
+CrateDB also comes with a `web administration interface`_ served on port
+``4200``.
 
-`Docker`_ is a lightweigt runtime to run distributed applications in a
-sandboxed container. We provide a `Docker image`_ for CrateDB for easy
-installation::
+Visit the admin interface in your browser at a URL like this one::
 
-    sh$ docker pull crate
-    sh$ docker run -d -p 4200:4200 -p 4300:4300 crate \
-          crate -Ccluster.name=my_cluster \
-                -Cnetwork.host=_site_
+  http://localhost:4200/
 
-.. _install_production:
 
-Install CrateDB for Production
-==============================
+.. NOTE::
 
-We provide distributions for **RHEL/YUM** based systems, **Ubuntu**, **Debian**
-and **Arch Linux**.
+    The admin interface runs on each CrateDB node.
 
-You can find instruction how to install CrateDB for production on the Download_
-page.
+.. NOTE::
 
-In case your distribution is missing feel free to `contact us
-<https://crate.io/about/contact/>`_. We're happy to create and maintain
-additional packages if we see enough interest.
+    If port ``4200`` is accessed via libraries or the command line, the request
+    will be handled by `the CrateDB Rest API`_, and the response will be JSON.
 
 .. SEEALSO::
 
- :doc:`Configuring CrateDB <configuration>`
-
- `Hello CrateDB`_ - To learn how to interact with CrateDB.
+ `An introductory tutorial`_
 
- `Multi Node Setup`_ - To see how to install and run CrateDB in a Multi Node
- setup.
+ `How to run CrateDB in a multi node setup`_
```

Comment:
the first thing we do at the top of the doc is link through to the install section of the guide, which includes the docker instructions

is this sufficient, or do you think we should call it out again?